dashboard/app/models/lesson.rb

```diff
@@ -484,6 +484,21 @@ def summarize_for_rollup(user)
     }
   end
 
+  def summarize_for_lesson_materials(user)
+    {
+      id: id,
+      unit: script.summarize_for_lesson_show,
+      position: relative_position,
+      key: key,
+      duration: total_lesson_duration,
+      name: localized_name,
+      resources: resources_for_lesson_plan(user&.verified_instructor?),
+      lessonPlanPdfUrl: lesson_plan_pdf_url,
+      lessonPlanHtmlUrl: lesson_plan_html_url,
+      scriptResourcesPdfUrl: script.get_unit_resources_pdf_url
```

Comment:
Several of these items seem not to be used (yet) on the front end. We can always go back and add them in when necessary, but for now let's keep this API response down to only the minimum set of what's actually used.

Reply:
I removed `duration` here but all the rest will be used in the very near future - unit, position, id, name, and resources are all used in this PR. 

- `lessonPlanPdfUrl` and `lessonPlanHtmlUrl` will be used in the resource list in this new `LessonMaterials` page.
- `scriptResourcesPdfUrl` will be used in the top level action button for unit-wide resources. 

These will all be used next week during the hackathon (or even tomorrow if I get to it).  Since these are pretty concretely tied to core planned tasks for this page, I personally feel pretty good about leaving them in given the short timeframe.  Thoughts?

Reply:
Sounds reasonable!